src/plugin/systems/joint.rs

```diff
@@ -39,7 +58,18 @@ pub fn init_joints(
         }
     }
 
-    for (entity, joint) in multibody_joints.iter() {
+    for (entity, entity_context_link, joint) in multibody_joints.iter() {
+        let context_entity = entity_context_link.map_or_else(
+            || {
+                let context_entity = try_get_default_context(&default_context).unwrap();
```

Comment:
unwrap -> expect

Reply:
I changed all these functions to panicking getters ; for consistency with `bevy::Query::get_single()`